kyverno/allow-host-namespaces.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: allow-host-namespaces
+  namespace: openebs
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Host namespaces (Process ID namespace, Inter-Process Communication namespace, and
+      network namespace) allow access to shared information and can be used to elevate
+      privileges. Pods should  allowed access to host namespaces.      
+spec:
+  validationFailureAction: audit
+  background: true
+  rules:
+    - name: host-namespaces
+      match:
+        resources:
+          name: "openebs-*"
+          kinds:
+            - Pod
+      validate:
+        message: >-
+          Sharing the host namespaces is allowed. The fields spec.hostNetwork,
+          spec.hostIPC, and spec.hostPID must be set to true.          
+        pattern:
+          spec:
+            =(hostPID): "true"
+            =(hostIPC): "true"
```

Comment:
Any components which use hostIPC? I am not sure of it. cc: @prateekpandey14

Reply:
I don't think as well, mostly not required to set.  its may be added here following the [standard template](https://kyverno.io/policies/pod-security/baseline/disallow-host-namespaces/disallow-host-namespaces/)